### PR TITLE
Add description to the call and extract it from the canvas

### LINKED ISSUE #5469: [Composer] Add default description
When doing the deploy call to the order API, atm there's no default description. This results in having an order in the table that has no readable description. 

Suggested description: "Requested with Instance Composer"

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

It's bundled as it was targeting the same piece of code

closes #5469 #5310 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
~~- [] Sufficient test cases (reproduces the bug/tests the requested feature)~~
~~- [ ] Correct, in line with design~~
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: src/UI/Components/Diagram/api/orderRequest.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { words } from "@/UI/words";
+import { InstanceForApi } from "../interfaces";
+
+export const sendOrder = (
```
Comment (LukasStordeur) [inline src/UI/Components/Diagram/api/orderRequest.ts:4]: PostOrder (based on the semantics of API requests)

Can you add docstrings?

**Review by LukasStordeur — COMMENTED** [review]
Could you add a test case for the method? sendOrder/postOrder

File: changelogs/unreleased/5469-composer-description.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+description: "Add description to the order api request and extract the call from the Canvas component"
+issue-nr: 5469
+change-type: patch
+destination-branches: [master, iso7]
```
Comment (LukasStordeur) [inline changelogs/unreleased/5469-composer-description.yml:4]: missing section

Comment (matborowczyk) [reply changelogs/unreleased/5469-composer-description.yml:4]: I removed description from that as that doesn't influence end-user

Comment (LukasStordeur) [reply changelogs/unreleased/5469-composer-description.yml:4]: It doesn't always need to affect the end user to be part of the changelog. The lack of a description was visible in the order view.

**Review by LukasStordeur — APPROVED** [review]

**Pull request closed without merging**